packages/imperative/__tests__/__integration__/cmd/__tests__/integration/cli/profiles/__scripts__/profiles/banana_profile_and_specify_cli.sh

```diff
@@ -1,25 +1,27 @@
-#!/bin/bash
+#!/bin/sh
 
-color=$1
-description=$2
-moldtype=$3
+profileColor=$1
+profileDescription=$2
+profileMoldType=$3
```

Comment:
I know most of these scripts are somewhat specific to the test files that call them, but should we do the same and add some information about the field in case it is missing?
`${1:?"First parm (profileColor) is required."}`

Reply:
All of these parameters were already defined in the script when I started work on this script, so I did not think to add the extra parameter testing.

I will be updating this script for the exitOnFailure changes, so I will add parameter tests at the same time.

By the way, the syntax for these parameters is not just human readable information. If the parameter is missing, the text will be displayed on standard error, and the script will terminate immediately. When our Jest scripts test for  standard error to be an empty string, the error about the missing parameter will be displayed in the Jest error report.